### PR TITLE
feat(indexer): Add processor to docker-compose

### PR DESCRIPTION
**Pull request by roninjin10**
- Add processor to docker-compose
- this is still a wip

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
                * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                  * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a> 
                    * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952?utm_source=stack-comment).

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
                * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                  * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a> 
                    * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952?utm_source=stack-comment).

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5942** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5942" target="_blank">Graphite</a> 
    * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
      * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
        * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
          * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
            * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
              * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
                * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                  * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a> 
                    * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952?utm_source=stack-comment).

**Review by hamdiallam — APPROVED** [review]

File: indexer/docker-compose.yml [hunk 1]
```diff
@@ -16,6 +16,29 @@ services:
     volumes:
       - postgres_data:/data/postgres
 
+  processor:
```
Comment (hamdiallam) [inline indexer/docker-compose.yml]: can we call this just `indexer`?

Comment (roninjin10) [reply indexer/docker-compose.yml]:
```suggestion
  indexer:
```

**Pull request merged**